components/primitives/index.js

```diff
@@ -3,5 +3,6 @@ import styled from 'styled-components';
 const Text = styled.Text``;
 const View = styled.View``;
 const Image = styled.Image``;
+const div = styled.div``;
```

Comment:
The idea of the primitives components are to be React Native compatible for the future. Using `div` escapes that. The `View` component (`styled.View`) is being mapped to `styled.div` behind the scenes, so is the same thing (should probably have `display: flex` by default though to work similar to React Native.

Reply:
This shouldn't break anything, since I'm overriding the `styled` object to do `styled.div = styled.View` and `styled.View = styled.div` depending on the platform (web, Figma, React Native or Sketch, etc), but using `View` instead would be cleaner/simpler I think

Reply:
Ah great, yes that's definitely cleaner. I've amended it to View. Thanks for your help!